src/org/usfirst/frc/team2706/robot/OI.java

```diff
@@ -47,6 +52,14 @@ public OI(Joystick driverStick, Joystick controlStick) {
         EJoystickButton backLeftButton = new EJoystickButton(driverStick, 5);
         backLeftButton.runWhileHeld(new AlignAndDistance(24));
 
+      EJoystickButton a = new EJoystickButton(driverStick, 1);
```

Comment:
You can auto-format code by pressing Ctrl + Shift + F in the file you want to format.